Overview.src.html

```diff
@@ -3944,11 +4021,22 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <span title=concept-request-body>body</span> is non-null, run these substeps:
 
   <ol>
+   <li><p>Let <var>stream</var> be an <span title=concept-empty-ReadableStream>empty</span>
+   <code title=concept-ReadableStream>ReadableStream</code> object.
+
    <li><p>Set <var>input</var>'s <span title=concept-Request-request>request</span>'s
-   <span title=concept-request-body>body</span> to an empty byte stream.
+   <span title=concept-request-body>body</span> to a new <span title=concept-body>body</span> whose
+   <span title=concept-body-stream>stream</span> is <var>stream</var>.
 
-   <li><p>Set <var>input</var>'s
-   <span title=concept-Request-disturbed-flag>disturbed flag</span>.
+   <li>
```

Comment:
This step confuses me. If input has a request body, then we override it with an empty stream, then immediately get a reader for the empty stream and read all the bytes in it, then never release the reader?

Shouldn't it instead be the case that we consume the request body, or transfer it, or tee it, or something? I forget what behavior we landed on, but this behavior seems wrong...